source/features/releases-tab.tsx

```diff
@@ -58,7 +58,7 @@ async function addReleasesTab(): Promise<false | void> {
 	}
 
 	// Wait for the tab bar to be loaded
-	const repoNavigationBar = (await elementReady('.UnderlineNav-body'))!;
+	const repoNavigationBar = (await elementReady('ul.UnderlineNav-body'))!;
```

Comment:
We should reuse `underlineNavDropdownUl` if possible

Reply:
But even then, why is this feature querying the same element twice (line 85)? It needs to be updated.

Reply:
Yeah I think this feature is too old and needs some enhancement too.
I'll work this PR again and reopen this PR.

Reply:
Hmm I'm not seeing any glaring issues with this feature, I would not rewrite it. If your latest changes work well already, let's just drop line 85

Reply:
Actually this was wrong, they're two different elements. I send a new PR for further adjustments